Directory.Build.props

```diff
@@ -30,10 +30,13 @@
     <ObjRoot>$(EnlistmentRoot)\..\obj</ObjRoot>
     <ObjRoot>$([System.IO.Path]::GetFullPath( $(ObjRoot) ))</ObjRoot>
 
+    <PackagesDir>$(EnlistmentRoot)\..\packages</PackagesDir>
+    <PackagesDir>$([System.IO.Path]::GetFullPath( $(PackagesDir) ))</PackagesDir>
+
     <RelativeOutputPathBase>$(MSBuildProjectDirectory.Substring($(EnlistmentRoot.Length)))</RelativeOutputPathBase>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
 
-    <OutputPath>$(BinRoot)\$(Configuration)\$(RelativeOutputPathBase)</OutputPath>
+    <OutputPath>$(BinRoot)\$(Configuration)\test\$(MSBuildProjectName)</OutputPath>
```

Comment:
I have a question about this.
I would expect properties in this file to apply to all projects.
But we wouldn't want all projects to output to a "test" directory.
Should this line be moved to the test.props instead?

Reply:
@TimothyMothra , this is the default path. Since we have product.props, they change the paths as well. in this first change i couldn't change and maintain both in the same .props file (the src and test folder)

Below an example:
```xml
  <PropertyGroup>
    <!--Removing the SRC folder from the output directory-->
    <CorePath>$(RelativeOutputPathBase)</CorePath>
    <OutputPath>$(BinRoot)\$(Configuration)\$(CorePath)</OutputPath>
    <OutputPath>$([System.IO.Path]::GetFullPath( $(OutputPath) ))\</OutputPath>
  </PropertyGroup>
```